update-extension-dependencies.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Purpose: Updates pom.xml files with symbolic dependencies to extensions to enforce a consistent build order.
+
+set -e -u -o pipefail
+shopt -s failglob
+
+echo ''
+echo 'Building bom-descriptor-json...'
+echo ''
+#mvn -D quickly -T0.8C -f independent-projects/arc
+#mvn -D quickly -T0.8C -f bom/application
+#mvn -D quickly -T0.8C -f core
+#mvn -D quickly -T0.8C -f extensions
```

Comment:
This and the previous commands would make sure that everything relevant that goes into the descriptor is up to date.
On the other hand we could instead just _expect_ everything to be up to date and remove those commands.

PS: Building everything _except_ `integration-tests` is not that easy, that's why multiple commands...

Reply:
Update: I removed those lines.